client/src/templates/Challenges/redux/completion-epic.js

```diff
@@ -62,16 +62,20 @@ function submitModern(type, state) {
     }
 
     if (type === actionTypes.submitChallenge) {
-      const { id } = challengeMetaSelector(state);
+      const { id, block } = challengeMetaSelector(state);
       const challengeFiles = challengeFilesSelector(state);
       const { username } = userSelector(state);
       const challengeInfo = {
         id,
-        files: challengeFiles.reduce(
+        files: undefined
```

Comment:
```suggestion
```

Reply:
This causes the line below to complain, because "property `files` does not exist on `challengeInfo`"

Reply:
Actually, this doesn't work because prettier.  There's no type issue though, because this is JS :man_shrugging: